getnacelle/nacelle-sanity-plugin-pim-linker
# feat: support clearing selected item + add nodemon

**What This Pr Does**
- adds button to allow clearing of selection
- make input editable for manually setting handles
- adds nodemon to watch and rebuild package

**How To Test**
- clone repo
- checkout branch `LS-1318--clear-selection`
- cd into repo and run `npm run build`
- run `npm link`
- from an existing sanity project run `npm link @nacelle/sanity-plugin-pim-linker`
- open up a piece of content that uses pim linker and you should see below screenshots

<img width="696" alt="Screen Shot 2021-11-12 at 10 40 57 AM" src="https://user-images.githubusercontent.com/24395931/141494279-0dc2738a-b63a-47a2-90de-79363f41ffb1.png">
<img width="690" alt="Screen Shot 2021-11-12 at 10 41 06 AM" src="https://user-images.githubusercontent.com/24395931/141494286-496b09c4-8a94-47bf-9032-8a96227f3627.png">
